server/opts.go

```diff
@@ -1714,7 +1761,12 @@ func parseJetStream(v interface{}, opts *Options, errors *[]error, warnings *[]e
 				opts.JetStreamMaxMemory = mv.(int64)
 				opts.maxMemSet = true
 			case "max_file_store", "max_file":
-				opts.JetStreamMaxStore = mv.(int64)
+				// need to define error to avoid non-name on left side of :=
+				var err error
```

Comment:
Also, I understand that you said that we could then use that other places, we could start with the `max_memory_store` just above, at least this configuration block would be consistent in ability to parse string byte sizes.

Reply:
sounds good. I didn't want to change everything and this not be an approach that was wanted. I'll change that one as well.